BPSampleApp/BPSampleAppTests/BPSampleAppTests.m

```diff
@@ -26,6 +26,10 @@ - (void)tearDown {
     [super tearDown];
 }
 
+void a_function_that_is_not_a_test() {
```

Comment:
Curious what is the purpose of this function since it is not called. Just to verify it won't be picked up as a test case?